pkg/kube/portforwarder.go

```diff
@@ -70,11 +70,11 @@ func (f *forwarder) Start() error {
 			fw, err := f.buildK8sPortForwarder(readyCh)
 			if err != nil {
 				errCh <- err
-				break
+				return
```

Comment:
I don't think I understand. It seems the difference between break and return here is to skip `readyCh = nil`.

Which seems fine, since the select should handle the errCh anyways. But wasn't sure what you meant by the goroutine hanging?

Reply:
Think about when `buildK8sPortForwarder` always fail

Reply:
then `break` will exit the for loop and the go routine will terminate? Just like return? Maybe I am missing something